include/swift/AST/DiagnosticsSema.h

```diff
@@ -35,6 +35,15 @@ namespace swift {
   // Declare common diagnostics objects with their appropriate types.
 #define DIAG(KIND,ID,Options,Text,Signature) \
     extern detail::DiagWithArguments<void Signature>::type ID;
+#define FIXIT(ID, Text, Signature)
+#include "DiagnosticsSema.def"
+  }
+
+  namespace fix_it {
+  // Declare common diagnostics objects with their appropriate types.
+#define DIAG(KIND,ID,Options,Text,Signature)
+#define FIXIT(ID,Text,Signature) \
+    extern detail::StructuredFixItWithArguments<void Signature>::type ID;
 #include "DiagnosticsSema.def"
```

Comment:
You could combine these two includes by pushing the namespace inside the macro…or by putting them both in `diag`.

Reply:
Now that they're in the same files, may as well have them share the diag namespace too.